wrappers/core18.go

```diff
@@ -586,6 +617,9 @@ func RemoveSnapdSnapServicesOnCore(s *snap.Info, inter interacter) error {
 
 	sysd := systemd.NewUnderRoot(dirs.GlobalRootDir, systemd.SystemMode, inter)
 
+	if err := undoSnapdDbusActivationOnCore(s); err != nil {
```

Comment:
is this bit tested (I haven't looked how core18_test.go is structured) ?

Reply:
I've added to some of the other existing tests to show that the D-Bus config files are not copied on classic systems (where code in `interfaces/dbus/backend.go` currently tries to handle this), and that they are removed when the snapd snap is uninstalled.